ROADMAP.md

```diff
@@ -1,34 +1,84 @@
 # v3.0 Road Map
 
 If you want to find out more about v3 we recommend you read the following issues
-  - [The future of FUI in 2019 and v3.0](https://github.com/fomantic/Fomantic-UI/issues/319)
-  - [v3 theming proposal](https://github.com/fomantic/Fomantic-UI/issues/441)
-  - [The discussion of jQuery & it's relevance to v3](https://github.com/fomantic/Fomantic-UI/issues/374)
+ - [The future of FUI in 2019 and v3.0](https://github.com/fomantic/Fomantic-UI/issues/319)
+ - [v3 theming proposal](https://github.com/fomantic/Fomantic-UI/issues/441)
+ - [The discussion of jQuery & it's relevance to v3](https://github.com/fomantic/Fomantic-UI/issues/374)
 
 > NOTE: Not all of these features/changes will be shipped with v3 but are on our roadmap unless specified otherwise.
 
+---
+
+### Key:
+
 | Key |                                                            |
 |-----|------------------------------------------------------------|
-| 💡  | Proposal                                                  |
-| ✏  | In planning phase                                         |
-| 🛠  | Currently being implemented                               |
+| 💡  | Proposal                                                   |
+| ✏  | In planning/prototyping phase                              |
+| 🛠  | Currently being implemented                                |
 | 🚫  | Blocked (progress can't continue until something is done) |
 | ✅  | Implemented and ready to ship                             |
 | ❌  | Canceled                                                  |
-| 🛳  | Must be done to ship v3                                   |
-
-- ✏ New documentation website 🛳
-- ✏ New build system 🛳
-- ✏ Visual component tests
-- 💡 CI/CD process
-- ✏ Rewrite modules into ES6 with classes 🛳
-- ✏ Move to a [monorepo](https://en.wikipedia.org/wiki/Monorepo) with separate packages 🛳
-- 💡 CLI tool
-- 💡 Theme builder website
-- 💡 Theme/plugin package sharing website
-- ✏ Changing icon implementation from class based to HTML attribute based (`data-icon="user"`) 🛳
-- ✏ Library theming system 🛳
-- ✏ Move from LESS to SASS
-- 💡 v3 to v2 JavaScript pollyfill package
+| 🛳  | Will ship with initial v3 release                          |
+
+---
+
+### Major Changes:
+
+- New documentation website ✏🛳
+- New build system ✏🛳
+- Visual component tests ✏
+- CI/CD process ✏
+- Rewrite modules into ES6 with classes ✏🛳
+- Move to a [monorepo](https://en.wikipedia.org/wiki/Monorepo) with separate packages ✏🛳
+- CLI tool ✏
+- Theme builder website 💡
+- Theme/plugin package sharing website 💡
+- Changing icon implementation from class based to HTML attribute based (`data-icon="user"`) ✏🛳
+- Theming implementation rewrite (behaviour will stay the same) ✏🛳
+- Move from LESS to SASS ✏🛳
+- v3 to v2 JavaScript pollyfill package 💡
+
```

Comment:
I suggest to add the following ideas for investigation (no promised features)
- A possible benefit when using CSS variables
```css
:root {
    --inverted-background: #1a1a1a;
}
div {
    background-color: var(--inverted-background);
}
```
- Support of IE11 only via 3rd party polyfills